bin/docker/kafka_bridge_config_generator.sh

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env bash
 
+if [ "$STRIMZI_TRACING" = "jaeger" ]; then
```

Comment:
I think we should check just empty string. Consider adding different type of tracing in the future, you would have to change this script too.

Reply:
good point, even because the bridge checks that the value is "jaeger" actually for checking the right env vars.